### ISSUE TITLE
Matrix with used psutil version for each build of NCPA

### ISSUE THREAD

**Issue by ccztux** [NagiosEnterprises/ncpa#843]
It would be great to have a matrix to see the used `psutil` version for each build of `NCPA`.

**Comment by MrPippin66** [comment]
Franky, I'd prefer the info be included in "-V" option for "ncpa_listener" or "ncpa_passive"

**Comment by jomann09** [comment]
Yeah currently we use whatever is the latest psutils during build, we could possibly add the psutils version to the output of the version command in NCPA 3 (since that's what we are working on atm)

**Comment by MrPippin66** [comment]
FYI, would be nice to also include that in the "agent_version" node information.

**Comment by jomann09** [comment]
That'd be easy enough as well, so we can plan on that as well

**Comment by MrPippin66** [comment]
@jomann09 I can submit this as a different feature request, but would it be possible to include the openssl version, as well, since I recently discovered it looks like openssl was statically linked in the executable?

**Comment by jomann09** [comment]
You don't need to open another request, when we work on this one we can also add that to the info

**Comment by gittethis** [comment]
hi,
i just realized the version i'm building is reporting strange (over 100% and some negatives) percent values on memory/swap check on a win2016 server (but not on the building box).
I tracked a bit the issue as being psutil's "fault", as in the deps the last version is indeed downloaded/used.
Can you tell me what psutil version was/is used to build 2.4.0 (the official release? From the .pyc files in the library zip i can't really tell)

Thanks!
